env_config/env.docker

```diff
@@ -194,3 +194,10 @@ GCNOTIFY_SMS_URL=https://api.notification.canada.ca/v2/notifications/sms
 # project to match the given user identifier. If left blank, defaults to the
 # user with system user ID 1.
 PROJECT_SEEDER_USER_IDENTIFIER=
+
+# Sets the number of desired seed projects to generate. Defaults to 1.
+NUM_SEED_PROJECTS=500
+
+# Sets the number of desired seed surveys to generate per project.
+# Defaults to 1.
+NUM_SEED_SURVEYS_PER_PROJECT=60
```

Comment:
I would change these defaults to something smaller, since the average person (especially if they aren't as savvy with these env files) probably doesn't want to seed this much stuff every time.

Maybe like 2 projects with 2 surveys each or something.

Reply:
Oops, yeah this is from testing lol. Two and two sounds good